classes/class3/catHW/public/rand.js

```diff
@@ -0,0 +1,21 @@
+function randInt(length) {
```

Comment:
It'd be perfectly acceptable to place it into your `/routes` folder, or if you have a lot of non-route utility files, make a `/util` directory. Just avoid throwing server logic into publicly accessible directories.